scripts/fetch-docs.sh

```diff
@@ -14,7 +12,7 @@ git clone --depth 1 --filter=blob:none --sparse https://github.com/openembedded/
 cd bitbake
 git sparse-checkout set doc/bitbake-user-manual/
 git fetch origin
-git checkout $BITBAKE_DOCS_COMMIT
+git checkout HEAD
```

Comment:
Looks like something that will make our tests unstable, since we won't know anymore to which version the texts refer

Reply:
Yes we should keep a hardcoded revision that can be updated by an automated PR. Only update it for major releases though (have a look at the tags history to determine when to trigger).

Reply:
If the tests are broken because of the update, then it means the docs format changed in some ways. Our tests should be changed accordingly. 

But I can make it to update on major releases.

Reply:
@deribaucourt There are tons of tags in the yocto-docs repo, which one do I check?

Reply:
I assume it is one starts with the `yocto`. Take a look at the new commits.

Reply:
Exactly. Make sure to not trigger this more than once a month though.

Reply:
I make it run on the 1st of every month.